app_src/ui/screens/camera_screen.py

```
@@ -2,345 +2,138 @@
+    @mainthread
+    def callable_test(self, result):
+        print(f"callable_test from App._ret_scan_load_config {result}")
+        # callable_test from App._ret_scan_load_config (1, 'QrWork::Timeout / Users ends')
+        # callable_test from App._ret_scan_load_config (0, '/data/user/0/org.wally.waller/cache/captures/IMG_20260414_225638.jpg')
+        print(f"got: {result}, [0]: {result[0]}")
+        if not result[0]:
+            complete_file_path = result[1]
+            print(f"using file: {complete_file_path}")
+            try:
+                self.app.file_operation.copy_add([complete_file_path])
+            except Exception as error_adding_image:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)